docs/source/migrating_tips.rst

```diff
@@ -1,83 +0,0 @@
-    ``/variants/import`` is now ``/variantsets/<variantSetId>/importVariants`` and 
-    ``/variants/export`` is ``/variantsets/<variantSetId>/export``.
-    These affect the generated client libraries slightly.
-
-Readsets/Readgroupsets
-  * ``readset`` has now been renamed to ``readgroupset``. This is mostly a straightforward replacement of the term. 
-  * ``readset.fileData[0].fileUri`` is now ``readgroupset.filename``
-  * ``readset.fileData[0].refSequences`` is replaced by ``readgroupset.referenceSetId``
-  * The rest of the ``readset.fileData`` field has been replaced by information within 
-    the ``readgroupset.readgroups`` array.
-  
-Reads
-  * All read positions are now 0-based longs, just like the variant positions.
-  * ``originalBases`` is now ``alignedSequence``
-  * ``alignedBases`` (``originalBases`` with the cigar applied) has been removed
-  * ``baseQuality`` is now an int array called ``alignedQuality``. You no longer 
-    need to subtract 33 or deal with ASCII conversion.
-  * ``name`` is now ``fragmentName``
-  * ``templateLength`` is now ``fragmentLength``
-  * ``tags`` is now ``info``
-  * ``position`` is now ``alignment.position.position``. The alignment object now contains
-    all alignment-related information - including the cigar, reference name, 
-    and whether the read is on the reverse strand.
-  * The old ``cigar`` string is now the structured field ``alignment.cigar``. To get 
-    an old-style cigar string, iterate over each element in the array, and
-    concat the ``operationLength`` with a mapped version of ``operation``. pseudocode::
-    
-      cigar_enums = {ALIGNMENT_MATCH: "M", CLIP_HARD: "H", CLIP_SOFT: "S", DELETE: "D",
-          INSERT: "I", PAD: "P", SEQUENCE_MATCH: "=", SEQUENCE_MISMATCH: "X", SKIP: "N"}
-
-      cigar_string = [c.operationLength + cigar_enums[c.operation] for c in read.alignment.cigar].join('')
-     
-     
-  * The old ``flags`` integer is now represented by many different first class fields.
-    To reconstruct a flags value, you need code similar to this pseudocode::
-    
-      flags = 0
-      flags += read.numberReads == 2 ? 1 : 0 #read_paired
-      flags += read.properPlacement ? 2 : 0 #read_proper_pair
-      flags += read.alignment == null ? 4 : 0 #read_unmapped
-      flags += read.nextMatePosition == null ? 8 : 0 #mate_unmapped
-      flags += read.alignment.position.reverseStrand ? 16 : 0 #read_reverse_strand
-      flags += read.nextMatePosition.reverseStrand ? 32 : 0 #mate_reverse_strand
-      flags += read.readNumber == 0 ? 64 : 0 #first_in_pair
-      flags += read.readNumber == 1 ? 128 : 0 #second_in_pair
-      flags += read.secondaryAlignment ? 256 : 0 #secondary_alignment
-      flags += read.failedVendorQualityChecks ? 512 : 0 #failed_quality_check
-      flags += read.duplicateFragment ? 1024 : 0 #duplicate_read
-      flags += read.supplementaryAlignment ? 2048 : 0 #supplementary_alignment
```

Comment:
I think the flags are still a useful to have in the documentation, since most bioinformaticians today are more comfortable with flags.  Probably a table denoting the new first class fields matching to a flag would also be helpful to guide and transition users.